graphics/macgui/macfontmanager.cpp

```diff
@@ -513,8 +514,10 @@ const Font *MacFontManager::getFont(MacFont *macFont) {
 			if (pFont != _uniFonts.end()) {
 				font = pFont->_value;
 			} else {
-				font = Graphics::loadTTFFontFromArchive("FreeSans.ttf", macFont->getSize(), Graphics::kTTFSizeModeCharacter, 0, 0, Graphics::kTTFRenderModeMonochrome);
```

Comment:
So, now you're breaking the existing code. That is bad.

What is going to happen if no font map is provided?

Reply:
So… are you suggesting that I go back and write code into the non-unicode branch？

Reply:
done the changes